index.html

```diff
@@ -1128,7 +1128,8 @@ <h2>Core Vocabulary Definitions</h2>
 <tr class="rfc2119-table-assertion" id="td-vocab-securityDefinitions--Thing"><td><code>securityDefinitions</code></td><td>Set of named security configurations
                 (definitions only). Not actually applied unless
                 names are used in a <code>security</code>
-                name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr></tbody></table><p><span class="rfc2119-assertion" id=
+                name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-profile--Thing"><td><code>profile</code></td><td>Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation.</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a> or <a>Array</a> of <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a></td></tr></tbody></table><p><span class="rfc2119-assertion" id=
```

Comment:
Nit: The profile member doesn't indicate a profile mechanism followed by a Thing Description, it _is_ a mechanism inside a Thing Description to indicate that the Web Thing it describes conforms to a particular profile or profiles.

Therefore I probably would have worded this slightly differently, e.g. "Indicates a profile or list of profiles to which the Web Thing conforms". Ideally with a link to the WoT Profile specification, if we can do that at this point.